tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -1087,6 +1091,47 @@ def distill(self, train_data=None, tuning_data=None, augmentation_data=None, tim
                                      verbosity=verbosity, models_name_suffix=models_name_suffix, teacher_preds=teacher_preds,
                                      augmentation_data=augmentation_data, augment_method=augment_method, augment_args=augment_args)
 
+    def plot_ensemble_model(self):
+        """
+            Output the visualized stack ensemble architecture of a model trained by `fit()`. 
+            The plot is stored to a file, `ensemble_model.png` in folder `Predictor.output_directory` 
+
+            This function requires `graphviz` and `pygraphviz` to be installed because this visualization depends on those package.
+            Unless this function will raise `ImportError` without being able to generate the visual of the ensemble model.
+
+        """
+        try:
+            import pygraphviz
+        except:
+            raise ImportError('Visualizing ensemble network architecture requires pygraphviz library')
+            
+        G = self._trainer.model_graph
+        remove = [node for node,degree in dict(G.degree()).items() if degree < 1]
```

Comment:
Why remove these nodes? What if the model with the best test score isn't used by any other model or I simply want to see all the models which were trained?

Reply:
I removed models which are not used by the best model. Unless, we will have several isolated nodes in the visual. I think it will be better to provide an option like prune_nodes (True or False).